vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -239,7 +239,13 @@ enum Result {
   CONTINUE = 32,
   IN_PROGRESS= 33,
   REWARDS_OFF = 34,
-  AC_OFF = 35
+  AC_OFF = 35,
+  
+  DEVICE_LIMIT_REACHED = 36,
+  MISMATCHED_PROVIDER_ACCOUNTS = 37,
+  UPHOLD_BLOCKED_USER = 38,
+  UPHOLD_PENDING_USER = 39,
+  UPHOLD_RESTRICTED_USER = 40
```

Comment:
It seems odd to have provider-specific errors in here. Do we need the `UPHOLD_` portion of these identifiers?

Reply:
My original thought was that "blocked", "pending", and "restricted" aren't really well-defined outside of the Uphold-specific context in which they are generated. If, in the future, we develop general semantics for them (e.g. Gemini starts returning "blocked" status or something like that), we can make the identifiers more general at that point.

No particularly strong feelings either way though.

Reply:
Yeah, fine by me either way too really (especially if other providers don't currently support those abilities).